### ISSUE TITLE
Google home prints out no certified matter device after scanning QR Code

### ISSUE THREAD

**Issue by Stephan-4711** [FezVrasta/node-red-matter#5]
Hi,

as you see error above

I used your examples and and added one of my devices 
Debug log in node red shows:

15.9.2023, 10:11:02[node: Light 1](http://MYIP:1880/#)
msg : string[29]
"No server or aggregator found"
15.9.2023, 10:11:02[node: Light 2](http://MYIP:1880/#)
msg : string[29]
"No server or aggregator found"
15.9.2023, 10:11:02[node: Outlet 1](http://MYIP:1880/#)
msg : string[29]
"No server or aggregator found"

**Comment by FezVrasta** [comment]
Could you please delete your flow, and just add a single Matter Device Status node, associated to an aggregated device with its bridge and let me know if everything works?

**Comment by Stephan-4711** [comment]
after deleting everything in side the flow and just left one Matter device status node, there is another error message.
No connection to device possible (or something similar, I translated from german)

**Comment by FezVrasta** [comment]
No errors on Node-RED? Try enabling the `debug` log level, restart Node-RED, and try to pair, then post here the logs

**Comment by FezVrasta** [comment]
Thanks, these don't look complete though, they should be much longer.

**Comment by Stephan-4711** [comment]
It's the debug log and I restarted whole system after activation

**Comment by Apollon77** [comment]
Seems like the controller found the device basically but then do not continue the pairing. What controller you tried?

**Comment by Apollon77** [comment]
Ps: aahhhh google and not certified? Did you added the device details to the google dev console? Google requires this for non certified devices like the node red flow does. 
See https://github.com/project-chip/matter.js#google-home-ecosystem

(But the log should be longer in any case!)

@FezVrasta maybe please Link to th  we page vm also officially in repo.

**Comment by Stephan-4711** [comment]
Oh I didn't know that.
I'll try tomorrow or Sunday for Google and alexa. I have the required hardware. 
Did you had this link anywhere in your git? Then I'm sorry I did not see that.

**Comment by FezVrasta** [comment]
I'm not quite following what's going on. This project is still in its early days so I'm not sure it's suitable for people with no coding experience

**Comment by Stephan-4711** [comment]
I'm just missing a sample config inside the wiki and also if there is a way to add not every device it self.
As far I was understanding the matter parts it looks like it should be possible to add a "bridge" for all devices I like to add to google home?

Is there a way to bring back the sample configurations without reinstalling the node red matter package?

**Comment by FezVrasta** [comment]
You can import the example flow at any time https://flowfuse.com/blog/2023/03/3-quick-node-red-tips-5/#:~:text=Click%20on%20the%20%22Examples%22%20button,to%20your%20Node%2DRED%20workspace.

**Comment by Stephan-4711** [comment]
Thanks, thats working for me now.

I tried to add the "matter" device to google and alexa at the same time.
If using the Philips Hue Hub this is possible.
Node red Addon is me giving this output, after I connected to google home:

`This device is already paired, you won't be able to pair it with other controllers until you decommission it from the paired controller first.`

will that be chaged, that I can connect the matter devices to all Voice Controllers?
